README.md

```diff
@@ -8,7 +8,52 @@ The Python [OpenTelemetry](https://opentelemetry.io/) client.
 
 ## Installation
 
-## Usage
+This repo include multiple installable packages. The `opentelemetry-api`
+package includes abstract classes that comprise the OpenTelemetry API following
+[the
+specification](https://github.com/open-telemetry/opentelemetry-specification).
+The `opentelemetry-sdk` package is the reference implementation of the API.
+
+To install the API and SDK packages, fork or clone this repo and do an
+[editable
+install](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs)
+via `pip`:
+
+```sh
+pip install -e ./opentelemetry-api
+pip install -e ./opentelemetry-sdk
+```
+
+The
+[`ext/`](https://github.com/open-telemetry/opentelemetry-python/tree/master/ext)
+directory includes OpenTelemetry integration packages, which can be installed
+separately as:
+
+```sh
+pip install -e ./ext/opentelemetry-ext-{integration}
+```
+
+## Development
```

Comment:
I feel like this should move to contributing.md. But maybe this heading should exist, pointing to contributing.md again.

Reply:
If we want a "Development section here at all, it should be a subsection of Contributing (or at least come after that section).

Reply:
Moved in 31723bc.